modules/nf-core/mageck/mle/main.nf

```diff
@@ -20,7 +20,8 @@ process MAGECK_MLE {
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    prefix = meta.id ?: "${meta.treatment}_vs_${meta.treatment}"
```

Comment:
Hi no this didn't end up working, i had to use a global variable because the script block and the directives are different scopes (I asked in the nf-core slack)

Reply:
ok, good to know, thanks!